packages/k8s-cni/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor:
+# Maintainer:
+pkgname=k8s-cni
+pkgver="v0.6.0"
+pkgrel=0
+pkgdesc="Kubernetes CNI needed for network pods"
+url="https://kubernetes.io/"
+arch="all"
```

Comment:
Does this package only for `armhf` architecture?

Reply:
Yes, it's only for armhf.

Reply:
Then line 8 should be `arch="armhf"`